river/drift/retrain.py

```diff
@@ -99,3 +101,77 @@ def _unit_test_params(cls):
             "drift_detector": drift.binary.DDM(),
         }
         yield {"model": naive_bayes.GaussianNB(), "drift_detector": drift.binary.DDM()}
+
+
+class DriftRetrainingRegressor(base.Wrapper, base.Regressor):
+    """Drift retraining regressor.
+
+    This regressor is a wrapper for any regressor. It monitors the incoming data for concept
+    drifts in the model's predictions. In case a drift is detected, the model will be reset.
```

Comment:
Hmm, that is a good point. We can turn it into a utility class or function, but I'm not sure it will be used in many places.

A bit of history:

- The original ADWIN paper states that the algorithm works with data in the `[0, 1]` range. Anything different than that should be normalized to this range. That is why I introduced the empirical normalization rule when I coded part of ARF in skmultiflow.
- According to the authors (I discovered that much later), empirically speaking, ADWIN also works with data outside the `[0,1]` range. So, idk XD; better safe than sorry?

Reply:
Yes keep it, I'm sure it doesn't affect performance that much.